Dockerfile

```diff
@@ -1,56 +1,18 @@
-FROM redis:6-buster
-USER root 
+FROM node:16-alpine
 
-# api port
-EXPOSE 3301
-
-# update and install initial packages
-RUN apt-get update && apt-get upgrade -y && apt-get install -y \
-    ca-certificates \
-    curl \
-    apt-transport-https \
-    lsb-release \
-    gnupg \
-    git \
-    make \
-    cmake \
-    build-essential \
-    python \
-    g++
-
-# SSL Errors Exist in 6-buster. Let's ignore them for now...
-RUN echo "-k" > ~/.curlrc
-# add yarn repo
-RUN curl https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-
-# install nvm and node 13.8.0
-SHELL ["/bin/bash", "--login", "-c"]
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
-RUN nvm install 16.5.0
-
-
-# install yarn
-RUN apt-get update && apt-get install --no-install-recommends -y \
-  yarn
+RUN apk add gettext
```

Comment:
Please add `python3` and `build-base` as these are required for arm builds to succeed (bcrypt doesn't have an armv7 version for pis... unfortunately).